jit-persistence/src/main/java/org/dcsa/jit/persistence/entity/TransportCall.java

```diff
@@ -16,6 +33,7 @@
 @Setter(AccessLevel.PRIVATE)
 @Entity
 @Table(name = "transport_call")
+@Inheritance(strategy = InheritanceType.TABLE_PER_CLASS)
```

Comment:
We only have one `transport_call` table and no subclasses?

Reply:
So I can extend TransportCall to TransportCallWithTimestamps that reads the transport_call_with_timestamps view

Reply:
Otherwise I would have to duplicate the entire TransportCall entity in UI instead of extending it

Reply:
I thought we could do that with a JOIN now. Restructuring the entities in UI-Support is not an issue if it leads to better / more understandable code.

I am not sure which way you prefer, but I am with this or the JOIN based approach.